src/greyVest/Fonts.js

```diff
@@ -16,6 +16,10 @@ let Fonts = () => (
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
-  </div>
+    <link
```

Comment:
Just a note - perhaps we should rename this component since `Fonts` now has a non-font css file. Though I guess it doesn't matter much because it is internal and used by Globals/Root 🤔

Reply:
yeah, I'd rather leave this one as-is for now - mostly because the Style component is so large that if we combined them into a single Root it would just be like 90% css. I would like to combine them eventually though.

Worth noting is that we *are* exporting Fonts and GVStyle components in prod right now - if we're gonna be nixing those later, we should just make sure to stop exporting them as part of 2.0.